app/serializers/v2/moves_serializer.rb

```diff
@@ -38,6 +39,7 @@ class MovesSerializer
       to_location
       prison_transfer_reason
       supplier
+      important_events
```

Comment:
Do you have a sense of how this will perform? on production the moves table is ~200k and the events table is ~400k rows.

Might this lead to the same joining flags for multiple moves performance issue we had a couple of weeks back?

Reply:
It's really fast - adds two extra queries which we can't really avoid but no N+1's. We could get the move incident events with just one query, but the front end guys wanted the PER medical event to also be included which therefore means eager loading the PER to achieve that. That said the PER is also included pretty much the whole time anyway, so shouldn't make things worse.

Flags were bad because of the indirect N:N relationship to PER via framework responses, of which there are many. This is a much simpler relationship dealing with far less data (very few moves or PER's will have these incident or medical flags) so the relevant queries are fast.

Reply:
👍 jolly good